electron/logger/logger.ts

```diff
@@ -69,13 +76,18 @@ const getPrefixString = (prefix: LogPrefix) => {
 export function logDebug(
```

Comment:
Seeing all of the repeated code here, maybe we should have a base logging function that gets the level and prefix passed to it instead

Reply:
Yes, I can add a TODO comment to come back here in the future